tests/schema/test_get_extensions.py

```diff
@@ -29,32 +29,36 @@ def test_returns_empty_list_when_no_custom_directives():
 def test_returns_extension_passed_by_user():
     schema = strawberry.Schema(query=Query, extensions=[MyExtension])
 
-    assert schema.get_extensions() == [MyExtension]
+    assert isinstance(schema.get_extensions()[0], MyExtension)
```

Comment:
Previously the test also made sure the correct amount of extensions are returned.

```suggestion
    assert len(schema.get_extensions()) == 1
    assert isinstance(schema.get_extensions()[0], MyExtension)
```